repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl

```diff
@@ -2,14 +2,40 @@
 <#include "../../include/imports.ftl">
 <#include "../../include/page-meta-data.ftl">
 <#import "../macros/components.ftl" as hee>
+<#include "../utils/document-formats.ftl">
 
 <@hst.setBundle basename="uk.nhs.hee.web.global"/>
 
+<#macro docDetailBlockForDocLink docLink>
+    <@hst.link var="fileURL" hippobean=docLink>
+        <@hst.param name="forceDownload" value="true"/>
+    </@hst.link>
+    <#assign docType>${docLink.filename?keep_after_last(".")}</#assign>
+    <li>
+        <a class="nhsuk-resources__link" href="${fileURL}" title="${docLink.filename}  (opens in new window)">
+            ${docLink.filename?keep_before_last(".")} - <span class="nhsuk-resources__tag nhsuk-resources__${docType}">${getDocumentFormat(docLink.filename?keep_after_last("."))}</span>
```

Comment:
Perhaps, we should display a short version of the document format (e.g. ODF, DOCX, etc) like the one indicated in the figma ? Also, the figma includes both long and short version, could we check with Andy/Jon if it was a mistake ? Thanks!